index.js

```diff
@@ -67,7 +67,7 @@ function Primus(server, options) {
   // connect to the server.
   //
   this.spec = {
-    timeout: options.timeout,
+    pingInterval: options.pingInterval,
```

Comment:
At least for the first few releases of this major, how do you feel about a deprecation error thrown if `options.timeout` or `options.ping` are defined?